backend/src/asset/asset.service.ts

```diff
@@ -5,29 +5,79 @@ import { AssetDto } from './dto';
 
 @Injectable()
 export class AssetService {
-    constructor(
-        private prisma: PrismaService,
-    ){}
-    async addToWallet(user: User, asset: AssetDto){
-        const value = Number(asset.value)
-        const wallet = await this.prisma.wallet.findUnique({
+  constructor(
+    private prisma: PrismaService,
+    ) { }
```

Comment:
The indentation looks weird here... maybe something like this?
```suggestion
  constructor(
    private prisma: PrismaService,
  ) {}
```
or it could be inline
```suggestion
  constructor(private prisma: PrismaService) {}
```